plover/gui_qt/dictionary_editor.ui

```diff
@@ -164,7 +164,7 @@
  </widget>
  <layoutdefault spacing="5" margin="8"/>
  <resources>
-  <include location="resources/resources.qrc"/>
+  <include location="../../build/lib/plover/gui_qt/resources/resources.qrc"/>
```

Comment:
Can those changes be reverted so the resource file is found when the build has not been run?

```bash
sed -e 's,../../build/lib/plover/gui_qt/resources/resources.qrc,resources/resources.qrc,' -i plover/gui_qt/*.ui
```